README.md

```diff
@@ -8,7 +8,7 @@
 
 This module implements an instance of the IBM Cloud Databases for etcd service.
 
-:exclamation: The module does not support major version upgrades or updates to encryption and backup encryption keys. To upgrade the version, create another instance of Databases for etcd with the updated version.
```

Comment:
Perhaps more clarity on what is and what is not allowed is required? In addition to the major version exclamation consider adding something like:

:exclamation: The module only supports setting the encryption and backup encryption key CRNs on creation and no update support is available. The KMS manual or automatic key rotation may be used to change the key value and initiate the re-encryption of the deployment.

Reply:
> I believe the intention of this exclamation, was that changing/updating the major version OR key OR backup key would cause a delete and re-create. As of 1.66.0 provider this is still the case. So this module does not support `updates to the encryption and backup encryption keys.`

OK, got it. That means this module doesn't support upgrading software version or keys once they are set.  If a user needs an upgraded version, they need to create a new instance and migrate data from the old instance to the new one.